src/decorators/withResources/index.js

```diff
@@ -53,27 +51,7 @@ export function withResources(resources = rootResources, parent = undefined) {
           },
           {
             new(data) {
-              const newResource = createResource(
-                instance.__meta,
-                Object.assign({}, resource, {
-                  data,
-                  method: HTTP_VERB_POST,
-                  name: 'new',
-                }),
-                parent
-              )
-
-              const originalThen = newResource.then
-
-              newResource.then = function then(resolve) {
-                // TODO: Reject when validation fails
-
-                return resolve({
-                  create: () => new Promise(originalThen),
-                })
-              }
-
-              return newResource
+              return createResource(instance.__meta, resource, parent).new(data)
```

Comment:
We support ```SDK.appointments.new()``` but since ```SDK.appointments.update()``` isn't possible this can only cause confusion.

The alternative (which already exists) is to use ```SDK.appointments().new()``` & ```SDK.appointments().update()```.

I suggest we remove this feature (```SDK.appointments.new()```) to improve ease of use.

Reply:
I agree, let's remove it.